composer.json

```diff
@@ -39,7 +38,8 @@
         "phpunit/phpunit": "5.*",
         "jakub-onderka/php-parallel-lint": "*",
         "jakub-onderka/php-console-highlighter": "*",
-        "fojuth/readmegen": "1.*"
+        "fojuth/readmegen": "1.*",
+        "amenadiel/jpgraph": "^3.6"
```

Comment:
Any reason this is moved into the dev section?

Reply:
Maybe added when I have tested the code, i will rollback to original content.

Reply:
Yes, work in progress